setup.py

```diff
@@ -35,6 +35,7 @@ def get_version():
         "google-cloud-bigquery",
         "google-cloud-storage",
         "Jinja2",
+        "pyjnius",
```

Comment:
We'e so far kept pyjnius out of `setup.py` and `requirements.txt`, instead keeping it in `java-requirements.txt`. Would it be possible to keep that separation?

cc @relud

Reply:
yup, this is actually not necessary. removed.